frontend/src/sagas.js

```diff
@@ -1719,7 +1789,20 @@ export function* fetchNextWorkflowitemHistoryPageSaga({
 
 export function* downloadDocumentSaga({ projectId, subprojectId, workflowitemId, documentId, showLoading = true }) {
   yield execute(function*() {
-    yield callApi(api.downloadDocument, projectId, subprojectId, workflowitemId, documentId);
+    try {
+      yield callApi(api.downloadDocument, projectId, subprojectId, workflowitemId, documentId);
+    } catch (error) {
+      yield put({
+        type: SNACKBAR_MESSAGE,
+        message: error.message
+      });
+      yield put({
+        type: SHOW_SNACKBAR,
+        show: true,
+        isError: true,
+        isWarning: false
+      });
+    }
```

Comment:
This is a hacky way to show the red error snackbar if the download fails. Normalle, `handleError()` should be called but this requires refactoring of the storage-service error handling